chall04/sadawi.py

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/python3
+#Python 3.6.9
+
+import sys
+
+def error(str):
+	print("Error: " + str)
+	sys.exit()
+
+try:
+	i = int(input())
+except:
+	error("Input is not a digit.")
+	sys.exit()
+
+arr = []
+for j in range(i):
+	arr.append(list(input()))
```

Comment:
Can be written the pythonic way: 

```python
arr = [list(input()) for _ in range(i)]
```

Plus, you are not using `j` so you should move it to a blank identifier. `_`